p2p/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+//! The Tendermint P2P stack.
+
+#![forbid(unsafe_code)]
+#![deny(
+    warnings,
+    missing_docs,
```

Comment:
While definitely sensible to have, during ongoing development these impede quite a lot. A better strategy is to have these not be denies in here, but enforcing them in CI.